src/components/EvidenceTile.tsx

```diff
@@ -22,7 +22,7 @@ export const EvidenceTile: FunctionComponent<Props> = (props) => (
       </Heading>
       <p className={`lbh-body-s ${styles.meta}`}>
         {props.createdAt}
-        {props.purpose && <> with {props.purpose}</>}
+        {/* {props.purpose && <> with {props.purpose}</>} */}
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, for now it's not needed but after talking to Jaye we thought this might be useful at a later stage (to maybe include the new "reason" described here  https://hackney.atlassian.net/jira/software/projects/DES/boards/45?selectedIssue=DES-159)

Reply:
Makes sense